guides/v2.3/graphql/payment-methods/authorize-net.md

```diff
@@ -0,0 +1,87 @@
+---
+group: graphql
+title: Authorize.Net payment method
+---
+
+Accept.js is a JavaScript-based solution for sending secure payment data directly to Authorize.Net. The Accept JavaScript library intercepts the payment data before it is passed to Magento and submits it directly to Authorize.Net, which replaces it with a one-time-use token, or payment nonce. This payment nonce, which is returned by the JavaScript library, is used in place of payment data.
+
+## Authorize.Net workflow
+
+The following diagram shows the workflow for placing an order when Authorize.Net is the selected payment method.
+
+![Authorize.Net sequence diagram]({{page.baseurl}}/graphql/images/authorize-net.svg)
+
+1. The customer clicks on the **Place order** button. The embedded `Accept.js` library captures the payment data and submits it directly to Authorize.Net.
+
+2. Authorize.Net returns a payment nonce and order details.
+Client's browser then posts the nonce to Magento server along with all the other order information. The payment nonce expires after 24 hours.
+
+3. The client uses the [`setPaymentMethodOnCart`]({{page.baseurl}}/graphql/reference/quote-payment-method.html) mutation to send the payment nonce and the last four digits of the card to Magento.
+
+4. Magento returns a `Cart` object.
+
+5. The client runs the [`placeOrder`]({{page.baseurl}}/graphql/reference/quote-place-order.html) mutation, which creates an order in Magento and begins the authorization process.
+
+6. Magento sends a transaction request that includes the payment nonce. The nonce replaces the payment details provided in standard Authorize.Net API calls.
+
+7. Authorize.Net sends a transaction response.
+
+8. Magento creates an order and sends an order ID in response to the `placeOrder` mutation.
+
+## Additional payment information
+
+When you set the payment method to `authorizenet_acceptjs` in the [`setPaymentMethodOnCart`]({{page.baseurl}}/graphql/reference/quote-payment-method.html) mutation, the `additional_data` object must contain a `authorizenet_acceptjs` object.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`cc_last_4` | Int! | The last four digits of the credit or debit card
+`opaque_data_descriptor` | String! | Authorize.Net's description of the transaction request
+`opaque_data_value` | String! | The nonce returned by Authorize.Net
+
+## Example setPaymentMethodOnCart mutation
+
+The following example assigns the `authorizenet_acceptjs` payment method to the specified cart.
```

Comment:
Depending on the above comment, this could be reworded to:
```suggestion
The following example assigns the `authorizenet_acceptjs` payment method to the `setPaymentMethodOnCart` mutation.
```

Reply:
The original sentence was correct.